README.md

```diff
@@ -204,6 +204,15 @@ Phantoms will no longer spawn if the hostile mobcap is full. This is per player.
 - Required options: `true`, `false`
 - Categories: `FEATURE`, `CARPET-ADDONS-NOT-FOUND`
 
+### pickFlowersFromPots
```

Comment:
The rule name should probably be `alwaysPickFlowersFromPots` to distinguish it from the vanilla behaviour with an empty hand.